css/survey-task.styl

```diff
@@ -141,6 +141,7 @@ $survey-task-pill-button
 
   > *
     margin: 0.3em 0.6em
+    display: inline-table;
```

Comment:
I think if we delete this line, but add:
```
.survey-task-chooser-choice
  display: flex
  justify-content: flex-start
  align-items: center
```
It will look as intended in all browsers. I checked on Chrome, Firefox, Safari and Edge.